### DISCUSSION TITLE
NativeAOT direct static linking to SQLite3?

### DISCUSSION THREAD

**Discussion by stevefan1999-personal** [ericsink/SQLitePCL.raw#582] [Q&A]
I don't want to have any DLL in between and want to have all of the sqlite3 function into one executable. Is it possible to have this feature?

Related: https://github.com/dotnet/runtime/discussions/96937

**Comment by ericsink** [comment]
Theoretically this would work, I think, if we had a nuget package with static libraries.  I'd have to investigate the practical feasibility.
